variants/frontend-base/template.rb

```diff
@@ -149,6 +148,30 @@
 
 File.write("./package.json", JSON.generate(package_json))
 
+#fix js lint issues with generated defaults before linting runs
+gsub_file "babel.config.js",
+          "module.exports = function(api) {",
+          "module.exports = api => {",
+          force: true
```

Comment:
```suggestion
          "module.exports = api => {"
```

Looking over the documentation & source, I'm pretty sure that `force` doesn't actually do anything for `gsub_file`, and that they just don't valid the hash.

Reply:
oh yep you're right, I think I'd added it in there when I was having issues replacing the line and just didn't take it back out